lua/cmp_nvim_ultisnips/treesitter.lua

```diff
@@ -0,0 +1,43 @@
+local M = {}
+
+-- Check if treesitter is available
+local ok_parsers, ts_parsers = pcall(require, "nvim-treesitter.parsers")
+if not ok_parsers then
+  ts_parsers = nil
+end
+
+local ok_utils, ts_utils = pcall(require, "nvim-treesitter.ts_utils")
+if not ok_utils then
+	ts_utils = nil
+end
+
+local function get_ft_at_cursor()
+  if not ok_parsers or not ok_utils then
+    error("[cmp_nvim_ultisnips.treesitter] Please install the nvim-treesitter plugin to use this feature.")
```

Comment:
Better error message? "This feature" sounds a bit vague.

Reply:
I think we shouldn't abort the plugin if the client doesn't yet install nvim-treesitter. In order for our plugin to works, do you think nvim-treesitter is required? If yes, please mention it from README, if no then we just use `print` instead of `error`, then people would be aware of it

Reply:
Treesitter should definitely not be required to use this plugin. If we auto-detect if treesitter is installed (option 2 in my other reply), we can remove this error message entirely.